src/witan/app/handler.clj

```diff
@@ -92,10 +93,10 @@
                         :middlewares [cors-mw token-auth-mw]
                         (ok {:message "hello"}))
             (sweet/POST* "/user" []
-                           :body [login-details w/LoginDetails]
+                           :body [user User]
                            :middlewares [cors-mw]
                            :summary "sign "
-                           (signup login-details))
+                           (signup user))
```

Comment:
It would be nice to use the compojure-api `:return` field for outward validation as well, I think. Both for this and also `/login`